gammapy/estimators/utils.py

```diff
@@ -590,3 +592,74 @@ def get_rebinned_axis(fluxpoint, axis_name="energy", method=None, **kwargs):
         edges = np.append(edges_min, edges_max[-1])
         axis_new = MapAxis.from_edges(edges, name=axis_name, interp=ax.interp)
     return axis_new
+
+
+def get_joint_significance_maps(
+    estimator, datasets, energy_edges=None, sum_over_energy_groups=True
+):
+    """Computes correlated excess and significance for a set of datasets.
+    The significance computation assumes that the model contains
+    one degree of freedom per valid bin in energy in each dataset.
+    This method implemented here is valid under the assumption
+    that the TS in each independent bin follows a Chi2 distribution,
+    then the sum of the TS also follows a Chi2 distribution (with the sum of degree of freedom).
+
+    See, Zhen (2014): https://www.sciencedirect.com/science/article/abs/pii/S0167947313003204,
+    Lancaster (1961): https://onlinelibrary.wiley.com/doi/10.1111/j.1467-842X.1961.tb00058.x
+
+
+    Parameters
+    ----------
+    estimator : `~gammapy.estimator.ExcessMapEstimator`
+        Excess Map Estimator
+    dataset : `~gammapy.datasets.Datasets`
+        Datasets
+    energy_edges : list of `~astropy.units.Quantity`, optional
+        Edges of the target maps energy bins. The resulting bin edges won't be exactly equal to the input ones,
+        but rather the closest values to the energy axis edges of the parent dataset.
+        Default is None: apply the estimator in each energy bin of the parent dataset.
+        For further explanation see :ref:`estimators`.
+    sum_over_energy_groups : bool
```

Comment:
same as above

Reply:
This one was not defined on the ExcessMapEstimator, moved there instead.